src/ResourceManager/Network/Commands.Network/Cortex/VirtualWan/P2sVpnServerConfiguration/UpdateAzureRmVirtualWanP2sVpnServerConfigCommand.cs

```diff
@@ -0,0 +1,287 @@
+            HelpMessage = "The name of the parent VirtualWan this P2SVpnServerConfiguration needs to be associated with.")]
+        [ValidateNotNullOrEmpty]
+        public virtual string ParentResourceName { get; set; }
+
+        [Alias("ResourceName", "P2SVpnServerConfigurationName")]
+        [Parameter(
+            Mandatory = true,
+            ParameterSetName = CortexParameterSetNames.ByP2SVpnServerConfigurationName + P2SVpnServerConfigurationParameterSets.Default,
+            HelpMessage = "The resource name.")]
+        [Parameter(
+            Mandatory = true,
+            ParameterSetName = CortexParameterSetNames.ByP2SVpnServerConfigurationName + P2SVpnServerConfigurationParameterSets.RadiusServerConfiguration,
+            HelpMessage = "The resource name.")]
+        [ValidateNotNullOrEmpty]
+        public virtual string Name { get; set; }
+
+        [Alias("P2SVpnServerConfigurationId")]
+        [Parameter(
+            Mandatory = true,
+            ValueFromPipelineByPropertyName = true,
+            ParameterSetName = CortexParameterSetNames.ByP2SVpnServerConfigurationResourceId + P2SVpnServerConfigurationParameterSets.Default,
+            HelpMessage = "The resource id of the P2SVpnServerConfiguration object to delete.")]
+        [Parameter(
+            Mandatory = true,
+            ValueFromPipelineByPropertyName = true,
+            ParameterSetName = CortexParameterSetNames.ByP2SVpnServerConfigurationResourceId + P2SVpnServerConfigurationParameterSets.RadiusServerConfiguration,
+            HelpMessage = "The resource id of the P2SVpnServerConfiguration object to delete.")]
+        [ValidateNotNullOrEmpty]
+        public string ResourceId { get; set; }
+
+        [Alias("P2SVpnServerConfiguration")]
+        [Parameter(
+            Mandatory = true,
+            ValueFromPipeline = true,
+            ParameterSetName = CortexParameterSetNames.ByP2SVpnServerConfigurationObject + P2SVpnServerConfigurationParameterSets.Default,
+            HelpMessage = "The P2SVpnServerConfiguration object to update.")]
+        [Parameter(
+            Mandatory = true,
+            ValueFromPipeline = true,
+            ParameterSetName = CortexParameterSetNames.ByP2SVpnServerConfigurationObject + P2SVpnServerConfigurationParameterSets.RadiusServerConfiguration,
+            HelpMessage = "The P2SVpnServerConfiguration object to update.")]
+        public PSP2SVpnServerConfiguration InputObject { get; set; }
+
+        [Parameter(
+            Mandatory = false,
+            HelpMessage = "The list of P2S VPN client tunneling protocols")]
+        [ValidateSet(
+            MNM.VpnGatewayTunnelingProtocol.IkeV2,
+            MNM.VpnGatewayTunnelingProtocol.OpenVPN)]
+        [ValidateNotNullOrEmpty]
+        public string[] VpnProtocol { get; set; }
+
+        [Parameter(
+            Mandatory = false,
+            ValueFromPipelineByPropertyName = true,
```

Comment:
Remove ValueFromPipelineByPropertyName from everything except ResourceId.

Reply:
Done.

Reply:
Ping on this, I still see it everywhere.